src/Forms/TextField.php

```diff
@@ -79,6 +79,17 @@ public function getAttributes()
         );
     }
 
+    public function getSchemaDataDefaults() {
```

Comment:
You'll need to drop the opening brace to a new line - see the linting failures on https://travis-ci.org/silverstripe/silverstripe-framework/jobs/311519327

Reply:
Thanks @robbieaverill. Will fix that.